OUXT-Polaris/robotx_packages
# Update to use latest vmrc

## Status
**Update to use latest vmrc**

## Description
Updated to use latest vmrc from osrf.
Following functions are added
- Use new usv_gazebo_plugin
- light buoy
- changeable marker

## Related PRs
No related PR.

## Related Branch
The following branches should be deleted after merging this PR.
- devel/feature/gazebo_thruster_plugin
- devel/feature/light_buoy
- devel/feature/robotx_core
- devel/feature/thruster_plugin

## Todos

## Steps to Test or Reproduce
You can check the two examples with following commands.

1. launch robotx wamv on ocean
```bash
$ roslaunch robotx_gazebo ocean.launch
```

2. launch robotx wamv with vmrc examples
```bash
$ roslaunch robotx_gazebo vmrc.launch
```

3. launch robotx_gazebo launcher that will also call detection, navigation programs
```bash
$ roslaunch robotx_gazebo robotx_gazebo.launch
```
